src/main/groovy/org/mockito/release/notes/vcs/IgnoredCommit.java

```diff
@@ -0,0 +1,11 @@
+package org.mockito.release.notes.vcs;
+
+import org.json.simple.Jsonable;
+import org.mockito.release.notes.model.Commit;
+
+import java.io.Serializable;
+
+public interface IgnoredCommit extends Jsonable, Serializable {
```

Comment:
Wow, that's pretty cool, I didn't know it's possible to cleanly serialize ignored commit predicate.

I suggest to make the code super simple and remove the need to serialize the predicate. You can [test for predicate here](https://github.com/mockito/mockito-release-tools/blob/b045b8fad5e1e9deceddc8de31c54d334a34ae65/src/main/groovy/org/mockito/release/notes/vcs/GitContributionsProvider.java#L44) and avoid keeping it as state in DefaultContributionSet.